var/spack/repos/builtin/packages/sirius/fj.patch

```diff
@@ -11,3 +11,15 @@ index 54a91df..ea66ecf 100644
                                                                                  type.gaunt_coefs().gaunt_vector(lm1, lm2));
                          hmt_[ia](j2, j1) = std::conj(hmt_[ia](j1, j2));
                      }
+diff -Nur SIRIUS-7.3.2/src/k_point/k_point_set.cpp SIRIUS-7.3.2-src/src/k_point/k_point_set.cpp
+--- SIRIUS-7.3.2/src/k_point/k_point_set.cpp	2022-07-05 16:19:54.000000000 +0900
++++ SIRIUS-7.3.2-src/src/k_point/k_point_set.cpp	2023-09-22 18:28:23.000000000 +0900
```

Comment:
Can you use a separate patch file or rename it? We keep track of a shasum -> file mapping, which breaks when modifying a file.

Reply:
@haampie  I apologize for the late response. I have created a separate patch file.